.evergreen/run-orchestration.sh

```diff
@@ -37,23 +37,34 @@ echo From shell `date` > $MONGO_ORCHESTRATION_HOME/server.log
 
 
 ORCHESTRATION_ARGUMENTS="-e default -f $MONGO_ORCHESTRATION_HOME/orchestration.config --socket-timeout-ms=60000 --bind=127.0.0.1 --enable-majority-read-concern"
-case "$OS" in
```

Comment:
`$OS` is the capitalized os name here so this never uses `virtualenv` on Windows https://evergreen.mongodb.com/task_log_raw/drivers_tools_tests_nossl__os_nossl~windows_64_vs2013_test_auth~auth_ssl~nossl_test_2.6_sharded_022b03e8a19ac6a8a41166c043d16e29f8245085_16_12_09_20_36_35/0?type=T#L459